### ISSUE TITLE
Intermittent test failure on TestHandleConnectionsCleanup()

### ISSUE THREAD

**Issue by laher** [oklog/oklog#54]
This part of the test seems to be failing under certain conditions: https://github.com/oklog/oklog/blob/master/pkg/ingest/conn_test.go#L68-L85

See Travis job https://travis-ci.org/oklog/oklog/jobs/218623755 for a failure. (It's on my branch, but I don't think my changes affect this test)

```
=== RUN   TestHandleConnectionsCleanup
--- FAIL: TestHandleConnectionsCleanup (1.06s)
	conn_test.go:75: HandleConnections successfully torn down (accept tcp [::]:43165: use of closed file or network connection)
	conn_test.go:84: timeout waiting for Close: initial Closes=0, current Closes=0
...
...

The command "go test -v -race ./{cmd,pkg}/..." exited with 1.
```

**Comment by peterbourgon** [comment]
Ugh, I should have fixed that. Stupid Travis. Thanks for the report.

**Comment by peterbourgon** [comment]
This was fixed as of v0.3.0. Thanks for the report.
